app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java

```diff
@@ -0,0 +1,88 @@
+package ar.com.wolox.android.example.ui.login;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.widget.EditText;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.inject.Inject;
+import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+
+/**
+ * LoginPresenter
+ */
+
+public class LoginPresenter extends BasePresenter<ILoginView> {
+
+    final String emailReq = "Email is required";
```

Comment:
This `emailReq`, `passReq`, `invEmail` (with better names 👀) was okey inside the `strings.xml`!